.github/workflows/run-phpunit.yml

```diff
@@ -0,0 +1,38 @@
+name: PHPUnit tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  phpstan:
+    runs-on: ubuntu-latest
```

Comment:
S tím latest bych byl opatrný. Dopoledne něco nasadíš a odpoledne už hotfix ne, protože budeš hledat proč ti padá pipeline, protože latest se změní na X.X verzi a bude tam bug. Raději bych upřednostnil fixnutou verzi.

Reply:
Changed to fixed version.